methods.py

```diff
@@ -906,21 +906,18 @@ def show_progress(env):
     node_count_fname = str(env.Dir("#")) + "/.scons_node_count"
 
     import math
-    import time
 
     class cache_progress:
-        # The default is 1 GB cache and 12 hours half life
-        def __init__(self, path=None, limit=1073741824, half_life=43200):
+        # The default is 1 GB cache
+        def __init__(self, path=None, limit=1073741824):
```

Comment:
```suggestion
        def __init__(self, path=None, limit=pow(1024, 3)):
```
Making magic numbers more readable.